### PR TITLE
Vote skip if >= 3

### PR DESCRIPTION
**Pull request by sdmway**
**Please describe the changes this PR makes and why it should be merged:**

- Added vote skip function when there is equal or more than 3 people in voice channel

### CODE REVIEW

**Comment by Assassin654** [comment]
make this toggleable.

**Comment by clownidze1337** [comment]
@Assassin654 
Done. Sorry for that delay 😮‍💨 
Was kinda busy...

**Comment by DarrenOfficial** [comment]
> make this toggleable.

why so rude

File: commands/togglevote.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  execute(message, args) {
+    if (voteskip) {
+      voteskip = false;
+      message.channel.send("Voteskip is now disabled!");
+    } else {
+      voteskip = true;
+      message.channel.send("Voteskip is now enabled!");
+    }
+  },
+};
```
Comment (DarrenOfficial) [inline commands/togglevote.js]: I think this is quite a rough way to add vote skip toggle feature. It's better if the way to toggle on & off is permission based.

**Review by DarrenOfficial — CHANGES_REQUESTED** [review]

**Comment by sdmway** [comment]
I could do it using admins ids from config file, but i decided that people may not want edit file often, so i did it with Discord Administrator permission.

**Comment by sdmway** [comment]
@DarrenOfficial i'll close this PR, and then i will create a new one to get things organized, i forgot about voteskip function in simple command, i did it only in slash

**Pull request closed without merging**